beam-postgres/examples/Pagila/Schema/V0001.hs

```diff
@@ -299,25 +299,33 @@ migration :: () -> Migration PgCommandSyntax (CheckedDatabaseSettings Postgres P
 migration () = do
 --  year_ <- createDomain "year" integer (check (\yr -> yr >=. 1901 &&. yr <=. 2155))
   PagilaDb <$> createTable "actor"
-                 (ActorT (field "actor_id" serial) (field "first_name" (varchar (Just 45)) notNull)
+                 (ActorT (field "actor_id" int)
```

Comment:
Do make sure that the migration in `Pagila/Schema/V0002.hs` can be used with this new type you've made.

Reply:
I was not able to load `Pagila/Schema/V0001.hs`
```
Could not find module ‘Pagila.Schema.V0001’
    Use -v to see a list of the files searched for.
```

I ran 
`stack repl --package beam-core --package beam-sqlite --package sqlite-simple` in the root of the project which dropped me into a repl. Inside `ghci`, I loaded `V0001.hs` and `V0002.hs`.
Any idea on how I can load it?